src/core/Surface/Surface.tsx

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { CommonProps, useTheme } from '../utils';
+import '@itwin/itwinui-css/css/surface.css';
+
+/**
+ * Helper function that returns one of the preset surface elevation values.
+ */
+const getSurfaceElevationValue = (elevation: SurfaceProps['elevation']) => {
+  if (!elevation) {
+    return '';
+  }
+
+  switch (elevation) {
+    case 4:
+      return '0 1px 10px rgba(0, 0, 0, 0.25)';
+    case 8:
+      return '0 3px 14px rgba(0, 0, 0, 0.25)';
+    case 16:
+      return '0 6px 30px rgba(0, 0, 0, 0.25)';
+    case 24:
+      return '0 9px 46px rgba(0, 0, 0, 0.25)';
+    default:
+      return '';
```

Comment:
Would there be valid case to have Surface without elevation? Then `elevation` could be undefined and make sense too :/

Reply:
Case has been added for elevation 0